src/tools/rustc-perf-wrapper/src/main.rs

```diff
@@ -163,7 +163,7 @@ fn apply_shared_opts(cmd: &mut Command, opts: &SharedOpts) {
     }
 }
 
-fn execute_benchmark(cmd: &mut Command, compiler: &PathBuf) {
+fn execute_benchmark(cmd: &mut Command, compiler: &Path) {
```

Comment:
@Integral-Tech , @compiler-errors , now I see error `cannot find type `Path` in this scope`, when I try to run `./x perf -- benchmark id`. Why this file doesn't get compiled in CI?

Reply:
@safinaskar Fixed in another PR #134816